src/main/java/com/google/devtools/build/lib/remote/ApiVersion.java

```diff
@@ -23,12 +23,18 @@ public class ApiVersion implements Comparable<ApiVersion> {
   public final int patch;
   public final String prerelease;
 
+  // The version of the Remote Execution API that Bazel supports initially.
+  public static final ApiVersion twoPointZero =
+      new ApiVersion(SemVer.newBuilder().setMajor(2).setMinor(0).build());
+  // The version of the Remote Execution API that starts supporting the
+  // Command.output_paths and ActionResult.output_symlinks fields.
+  public static final ApiVersion twoPointOne =
+      new ApiVersion(SemVer.newBuilder().setMajor(2).setMinor(1).build());
+
   // The current lowest/highest versions (inclusive) of the Remote Execution API that Bazel
   // supports. These fields will need to be updated together with all version changes.
-  public static final ApiVersion low =
-      new ApiVersion(SemVer.newBuilder().setMajor(2).setMinor(0).build());
-  public static final ApiVersion high =
-      new ApiVersion(SemVer.newBuilder().setMajor(2).setMinor(0).build());
+  public static final ApiVersion low = twoPointZero;
+  public static final ApiVersion high = twoPointOne;
```

Comment:
I don't think this is accurate. With this change applied, Bazel supports up to and including 2.3 just fine:

- We already send platform properties as part of the Action message, as required by REv2.2.
- We already rely on `argv[0]` to be resolved relative to the working directory, using `$PATH` lookups, as required by REv2.3.

So in that sense Bazel will behave more like an REv2.3 client than REv2.1 with this change applied.

Reply:
My plan was to finish this PR with 2.1 first, then a subsequent PR could raise it from 2.1 -> 2.3.
That would allow the change to be more atomic if things need to be reverted (which did happen).

I have been busy with more customer-facing tasks so this PR has been slow to iterate upon. 🐌
I hope I could spend a bit more time on this PR at the end of this week / early next week. 🤞

Reply:
Bumped to 2.2 since 2.3 tag in remote-apis I not yet created https://github.com/bazelbuild/remote-apis/tags